__tests__/app.test.js

```diff
@@ -69,5 +54,30 @@ describe("GET /api/topics", () => {
             })
         }))
     })
-
 })
+
+describe("GET /api", () => {
+    test("status: 200", () => { 
+        return request(app)
+        .get("/api")
+        .expect(200)
+
+    })
+    test("responds with an object describing all other endpoints available", () => { 
+        return request(app)
+        .get("/api")
+        .then(({body: {endpoints}}) => {
+            expect(typeof endpoints).toBe("object")
+        })
+    })
+    test("responds with an endpoints object, where each endpoint has a description property", () => {
+        return request(app)
+        .get("/api")
+        .then( async ({body: {endpoints}}) => {
+            const data = await fs.readFile("./endpoints.json")
+            const parsedData = JSON.parse(data)
+            expect(endpoints).toEqual(parsedData)
+        })
```

Comment:
Could have used require here instead, which may be easier as its synchronous 👍 Nothing wrong with this though 🙂